Code/GraphMol/Wrap/Conformer.cpp

```diff
@@ -28,6 +29,29 @@ RDGeom::Point3D GetAtomPos(const Conformer *conf, unsigned int aid) {
   return res;
 }
 
+PyObject* GetPos(const Conformer *conf) {
+    RDGeom::POINT3D_VECT pos = conf->getPositions();
```

Comment:
This sound be `const RDGeom::POINT3D_VECT &pos` in order to prevent all of the positions from having to be copied.